test/api/package/get-owned-packages.test.js

```diff
@@ -0,0 +1,94 @@
+const test = require('ava')
+
+test('stub', (t) => {
+  t.true(true)
```

Comment:
i don't think get-owned-packages belongs in this PR since the impl/tests aren't done

Reply:
i just moved the file - no changes were made